File: src/main/java/src/SistemaGestionEstudiante/Escuela.java
package SistemaGestionEstudiante;

import java.util.ArrayList;
import java.util.List;

public class Escuela {
    private List<Estudiante> estudiantes;
    private List<Curso> cursos;

    public Escuela() {
        estudiantes = new ArrayList<>();
        cursos = new ArrayList<>();
    }

    public void agregarEstudiante(Estudiante estudiante) {
        estudiantes.add(estudiante);
        System.out.println(estudiante.getNombre() + " ha sido agregado.");
    }

    public void agregarCurso(Curso curso) {
        cursos.add(curso);
        System.out.println("Curso " + curso.getNombre() + " ha sido agregado.");
    }

    public void mostrarEstudiantes() {
        System.out.println("Estudiantes:");
        for (Estudiante estudiante : estudiantes) {
            System.out.println(estudiante);
        }
    }

    public void mostrarCursos() {
        System.out.println("Cursos:");
        for (Curso curso : cursos) {
            System.out.println(curso);
        }
    }
}

File: src/main/kotlin/SistemaGestionEstudiante/Escuela.kt
package org.example.SistemaGestionEstudiante

class Escuela {
    private val estudiantes: MutableList<Estudiante> = mutableListOf()
    private val cursos: MutableList<Curso> = mutableListOf()

    fun agregarEstudiante(estudiante: Estudiante) {
        estudiantes.add(estudiante)
        println(estudiante.nombre + " ha sido agregado.")
    }

    fun agregarCurso(curso: Curso) {
        cursos.add(curso)
        println("Curso " + curso.nombre + " ha sido agregado.")
    }

    fun mostrarEstudiantes() {
        println("Estudiantes:")
        for (estudiante in estudiantes) {
            println(estudiante)
        }
    }

    fun mostrarCursos() {
        println("Cursos:")
        for (curso in cursos) {
            println(curso)
        }
    }
}